### PR TITLE
New release for Scorecard v4.8.0

### PR DESCRIPTION
**Pull request by naveensrinivasan**
- New release to address the issue https://github.com/ossf/scorecard/issues/2338

Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,25 +6,25 @@ require (
 	github.com/caarlos0/env/v6 v6.10.1
 	github.com/google/go-cmp v0.5.9
 	github.com/google/go-github/v46 v46.0.0
-	github.com/ossf/scorecard/v4 v4.7.0
+	github.com/ossf/scorecard/v4 v4.8.0
```
Comment (spencerschrock) [inline go.mod:9]: The Makefile needs to reflect this version bump too (and the new commit SHA)

**Review by spencerschrock — APPROVED** [review]
LGTM once the Makefile changes are added

File: go.mod (hunk 1 above)
Comment (naveensrinivasan) [reply go.mod:9]: @spencerschrock  Thanks; I have updated it. Can you please take a look?

**Review by spencerschrock — APPROVED** [review]

**Review by azeemshaikh38 — APPROVED** [review]
Thanks!

**Pull request merged**